bin/bootstrap-vagrant.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# If there is an error, exit early
+set -e
+
+# Update apt-get
+if ! test -f .updated-apt-get; then
+  sudo apt-get update
+  touch .updated-apt-get
+fi
+
+# Install pip
+if ! which pip &> /dev/null; then
+  sudo apt-get install -y python-setuptools python-pip
+fi
+
+# Upgrade pip
+if ! pip --version | grep 'pip 7.1.0' &> /dev/null; then
+  sudo pip install pip==7.1.0
+fi
+
+# Install git
+if ! which git &> /dev/null; then
+  sudo apt-get install -y git
+fi
+
+# Install PostgreSQL and psycopg2 dependencies
+if ! which psql &> /dev/null; then
+  # Setup PostgreSQL 9.3 for our database
+  sudo apt-get install -y postgresql-9.3 postgresql-server-dev-9.3 python-dev
+fi
+
+# Set up PostgreSQL 9.3 configuration
```

Comment:
Could use 9.4. Has some nice features. Better support for json, uuid (auto-gen uuids for ids, etc).